inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/file/FileReaderOperator.java

```diff
@@ -90,7 +93,9 @@ public Message read() {
                 AuditUtils.add(AuditUtils.AUDIT_ID_AGENT_READ_SUCCESS,
                         inlongGroupId, inlongStreamId, System.currentTimeMillis());
                 readerMetric.pluginReadCount.incrementAndGet();
-                return new DefaultMessage(message.getBytes(StandardCharsets.UTF_8));
+                Map<String, String> header = new HashMap<>();
+                header.put(PROXY_KEY_DATA, DigestUtils.md5Hex(inlongGroupId));
+                return new DefaultMessage(message.getBytes(StandardCharsets.UTF_8), header);
```

Comment:
Is it better to use inlongGroupId and inlongStreamId?

Reply:
@pocozh 3qs your advice, I think inlongGroupId more better becuse the Group ID of the same task does not change. moreover added that users can configure Partition keys.